src/cryptenroll/cryptenroll.c

```diff
@@ -486,6 +499,28 @@ static int prepare_luks(
                                 strlen(envpw));
                 if (r < 0)
                         return log_error_errno(r, "Password from environment variable $PASSWORD did not work.");
+        } else if(arg_unlock_file) {
```

Comment:
space between `if` and `(`...

More importantly though. I am pretty sure if the user specified the file on the cmdline, it should take precedence over the PASSWORD env var. I.e. do this stuff first.